docs/google.rst

```diff
@@ -155,6 +159,16 @@ Examples::
         """
     }
 
+    process allowing_some_series {
+        cpus 8
+        memory '20 GB'
+        machineType 'n2,c2,m3'
```

Comment:
I think wildcards should be used select machine type. I mean `'n2*,c2*,m3*'. This would allow more precise control on the types that can be used